cmd/kubelet/app/server.go

```diff
@@ -745,7 +747,8 @@ func makePodSourceConfig(kc *KubeletConfig) *config.PodConfig {
 type KubeletConfig struct {
 	Address                        net.IP
 	AllowPrivileged                bool
-	CadvisorInterface              cadvisor.Interface
+	CAdvisorInterface              cadvisor.Interface
+	CAdvisorProxyPort              uint
```

Comment:
I don't think this is used by anything any longer and can be removed from this struct.